docs/dev/dod.md

```diff
@@ -0,0 +1,14 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
+- Travis builds the project and runs all tests without error
```

Comment:
As discussed in the meeting on sunday, this might need some exceptions. Because some tests might not be able to be performed in the 50 minute timeframe that travis provides.

A external CI server like Jenkins might be a solution for this